browse/templates/cookies.html

```diff
@@ -0,0 +1,83 @@
+{%- extends "base.html" -%}
+
+{% block title %}Your cookie configuration{% endblock %}
+
+{% block header_h1 %}<h1>arxiv.org > cookies</h1>{% endblock header_h1 %}
+
+{%- block content %}
+<h1>Your cookie configuration for arXiv</h1>
+
+<p>You can use this form to set preferences that will apply to all arXiv.org sites for this
+and future sessions when access from the same machine. Cookies are snippets of information
+stored on your machine and returned to our site with each request you make (click
+show additional debugging information below to see what is stored). 
+This facility will not work if you have disabled cookies or on browsers that do not
+support cookies.</p>
+
+<p>For information about the full-text formats available, consult the
+<b>  TODO get help from some url_for like source </b>
```

Comment:
TODO in output

Reply:
How do I get a URL like /help/view from url_for?

Reply:
You would either have to explicitly define it in arxiv-base (https://github.com/arXiv/arxiv-base/blob/33fbca124d543de2e2fc7791fb75d06dc11fb2a7/arxiv/base/config.py#L16) or something like `"{{ url_for('help') }}/view"`.